src/features/embalm/stepContent/hooks/useCreateSarcophagus.ts

```diff
@@ -335,6 +335,7 @@ export function useCreateSarcophagus() {
 
   const handleCreate = useCallback(async () => {
     setCurrentStage(CreateSarcophagusStage.DIAL_ARCHAEOLOGISTS);
+    setStageError(undefined);
```

Comment:
The stage error should probably be `undefined` by the time this function is ever called. We may want to clear it on the stages component unmount. Doesn't hurt to have it here, so will merge as-is.